app/views/catalog/_document.html.erb

```diff
@@ -2,5 +2,6 @@
 <article class="document <%= render_document_class document %> document-position-<%= document_counter%> " data-document-counter="<%= document_counter %>" itemscope itemtype="<%= document.itemtype %>">
   <%= render_document_partials document,
     blacklight_config.view_config(document_index_view_type).partials,
-    document_counter: document_counter %>
+    document_counter: document_counter,
+    presenter: presenter(document, document_counter: document_counter) %>
```

Comment:
```suggestion
    presenter: presenter(document, counter: document_counter) %>
```

Reply:
I should revisit this in after a rebase, but the keyword param on the presenter class is `counter`, and we probably also want to use the same value from the DocumentComponent (which takes the search offset into account, if available). `document_counter` is relative to the current list.